src/main/webapp/js/components/entities/entity-table.jsx

```diff
@@ -103,7 +103,10 @@ class EntityTable extends Component {
               )}
               getTdProps={(state, rowInfo) => ({
                 onClick: () => {
-                  if (_.get(rowInfo, 'original.id') && this.canWrite()) {
+                  if (_.get(rowInfo, 'original.id')
+                  && this.canWrite()
+                  && _.get(rowInfo, 'original.name') !== 'Admin'
```

Comment:
why you added this change? admin user can be edited (only some of the fields should not be editable)